libs/runtime/ebpf_hash_table.c

```diff
@@ -758,7 +758,13 @@ ebpf_hash_table_create(_Out_ ebpf_hash_table_t** hash_table, _In_ const ebpf_has
     table->entry_count = 0;
     table->seed = ebpf_random_uint32();
     table->extract = options->extract_function;
+#if defined(NDEBUG)
```

Comment:
Please change comment in line 59.
What is the reason behind not maintaining entry count in retail? Performance?

Reply:
Correct. An global interlocked on each insert and remove adds up fast.